package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "eslint-config-fluid",
+    "version": "0.1.0",
+    "description": "fluid-project eslint configuration",
+    "main": "index.js",
+    "engines": {
+        "node": ">=4.0.0"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/fluid-project/eslint-config-fluid.git"
+    },
+    "keywords": [
+        "eslint",
+        "eslintconfig",
+        "fluid",
+        "fluid-project"
+    ],
+    "author": "Fluid Project",
+    "license": "BSD-3-Clause",
+    "bugs": {
+        "url": "https://issues.fluidproject.org/projects/FLUID/issues"
+    },
+    "homepage": "http://fluidproject.org",
+    "peerDependencies": {
+        "fluid-eslint": ">=2.0.0"
```

Comment:
eslint indicates that we should declare a peer dependency on it ( http://eslint.org/docs/developer-guide/shareable-configs#publishing-a-shareable-config ). This will cause NPM to throw a warning if the peer dependency is not installed. However, our projects do not actually depend on eslint, but rather a fork "fluid-eslint" instead. 

I agree that this module should work with either, although we haven't tested it with the main line of eslint. However, we still need to declare the peerDependency to ensure that it is present. Here's an article on peerDependencies ( https://nodejs.org/en/blog/npm/peer-dependencies/ ).

I pinged you in the channel last Friday to ask about what the difference between our fork and the mainline are, and if we could just switch back to use the main line of eslint now. ( https://botbot.me/freenode/fluid-work/2016-09-23/?msg=73496446&page=1 ). I think the ideal case will be to switch to use the mainline of eslint and not maintain a separate fork. If that's still not possible though, we'll likely either have to restrict this module to only work with our fork, or to remove the validation that the eslint/fluid-eslint is actually present.